### ISSUE TITLE
See how many people use a specific fetch field

### ISSUE THREAD

**Issue by eepykate** [unixporn/trup#146]
`top` lists the top 5, but it would be interesting to see the number of users who use a specific thing.

example usage

```
[user] !users Distro Fedora
[trup] 35 (1%)
---------
[user] !users DeWm GNOME
[trup] 50 (2%)
```

**Comment by mlvzk** [comment]
SQL query for future reference
```sql
SELECT
  count,
  ROUND((count::float/total::float)*100) AS percentage
FROM (
  SELECT
    count(*),
    (SELECT count(*) FROM sysinfo WHERE info->>'Distro' != '') AS total
  FROM sysinfo WHERE info->>'Distro' ~* 'fedora'
) t;
 count | percentage
-------+------------
     8 |          2
(1 row)
```

**Comment by Tanish2002** [comment]
I'll try to do this

**Comment by eepykate** [comment]
Don't forget to make sure it supports spaces 🙃

**Comment by Tanish2002** [comment]
@mlvzk can you give sql to get all the possible "Fields" i.e DeWm, Distro, etc
so if the person just does `!users` he can get a message with all the available fields

**Comment by eepykate** [comment]
fixed with #149
